charts/modular-loki/templates/querier/querier-hpa.yaml

```diff
@@ -19,13 +19,13 @@ spec:
   metrics:
     - type: Resource
       resource:
-        name: cpu
+        name: memory
```

Comment:
Seriously? Wow....

Reply:
Yeah. You might think they're scala style bags without order, but kubernetes does not agree.

Reply:
What is next, start ordering environment variables? I don't think the order ever mattered but I take your word for it.